source/includes/_05-one-time-sync-ekyc.html.md.erb

```diff
@@ -385,6 +385,32 @@ app.post("/checks", (req, res) => {
         PassFort to investigate issues that may arise with your integration.
       </td>
     </tr>
+    <tr>
+      <td><code>charges</code></td>
+      <td>
+        An array containing one or more
+        <a href="#check-charges">Charges</a>
+      </td>
+      <td>No</td>
+      <td>
+        A list of charges incurred whilst running this check. Charges may only
```

Comment:
North Americans get super confused about the word "whilst". We should use "while" wherever possible

Reply:
🤨

Reply:
I know, I know. But they really confused. Many of them don't know it's a word